yb-voyager/cmd/importData.go

```diff
@@ -436,28 +436,49 @@ func checkPrimaryKey(tableName string) bool {
 	}
 	defer conn.Close(context.Background())
 
-	var table, schema string
+	var table, schema, originalTableName string
 	if len(strings.Split(tableName, ".")) == 2 {
 		schema = strings.Split(tableName, ".")[0]
 		table = strings.Split(tableName, ".")[1]
 	} else {
 		schema = target.Schema
 		table = strings.Split(tableName, ".")[0]
 	}
+	originalTableName = table
 
-	if sourceDBType == ORACLE && !utils.IsQuotedString(table) {
+
+	if utils.IsQuotedString(table) {
+		table = strings.Trim(table, `"`)
+	  } else {
 		table = strings.ToLower(table)
+	  }
+
+	checkTableSql := fmt.Sprintf(`SELECT '%s.%s'::regclass;`, schema, originalTableName)
+	log.Infof("Running query on target DB: %s", checkTableSql)
+
+	rows, err := conn.Query(context.Background(), checkTableSql)
+	if err != nil {
+		utils.ErrExit("error in querying to check table %q is present: %s", table, err)
+	}
+	
+	if(rows.Next()){
```

Comment:
No need of parenthesis around expression. Setup your IDE/editor to automatically format the code using `go fmt`.